types/schemas/prototypeApplication/data/Proposal.ts

```diff
@@ -1,9 +1,303 @@
+    count?: {
+      bathrooms?: Integer;
+      bedrooms?: Integer;
+      dwellings?: Integer;
+    };
+  };
+  newDwellings?: {
+    newBuild?: {count: Integer};
+  };
+  units?: ResidentialUnits;
+  watercourse?: {
+    name: string;
+    type: 'ditch' | 'millStream' | 'pond' | 'river' | 'streamOrBrook' | 'other';
+  };
+  structures?: {
+    type:
+      | 'bridge'
+      | 'catchpit'
+      | 'culvert'
+      | 'pipe'
+      | 'gully'
+      | 'headwall'
+      | 'manhole'
+      | 'weir'
+      | 'other';
+    total: Integer;
+    permanent?: {
+      count: Integer;
+    };
+    temporary?: {
+      count: Integer;
+    };
+  };
+  environmentalImpactDescription?: string;
+}
+
+/**
+ * @description Proposal details for project sites within the Greater London Authority (GLA) area
+ */
+export interface LondonProposal extends Omit<UKProposal, 'units'> {
+  schemeName?: string;
+  /**
+   * @description Proposed parking spaces
+   */
+  parking?: {
+    cars?: ProposedCount;
+    vans?: ProposedCount;
+    motorcycles?: ProposedCount;
+    cycles?: ProposedCount;
+    buses?: ProposedCount;
+    disabled?: ProposedCount;
+    carClub?: ProposedCount;
+    offStreet?: {residential: ProposedCount};
+    other?: ProposedCount;
+  };
+  /**
+   * @description Creating new buildings
+   */
+  newBuildings?: NewBuildingsOrStoreys;
+  /**
+   * @description Increasing the height of existing buildings
+   */
+  newStoreys?: NewBuildingsOrStoreys;
+  /**
+   * @description Project cost
+   */
+  cost?: {
+    projected: '2m' | '2mTo100m' | '100m';
+  };
+  /**
+   * @description Electric vehicle charing points
+   */
+  charging?: {
+    active?: {count: Integer};
+    passive?: {count: Integer};
+  };
+  /**
+   * @description Changes that result in the loss, gain, or change of use of natural spaces
+   */
   nature?: {
     openSpaces?: {
+      impact: 'loss' | 'gain' | 'change';
+      description: string;
+      type: OpenSpaceType;
       designation: OpenSpaceDesignation;
+      access: 'restricted' | 'unrestricted';
+      area: {hectares: number};
+      /**
+       * @description Whether the open space change involves a land swap
+       */
+      swap: boolean;
+    }[];
+    protectedSpaces?: {
+      impact: 'loss' | 'gain' | 'change';
+      description: string;
+      designation: ProtectedSpaceDesignation;
+      access: 'restricted' | 'unrestricted';
+      area: {hectares: number};
+    }[];
+  };
+  /**
+   * @description Water management
+   */
+  water?: {
+    /**
+     * @description Internal residential water usage
+     */
+    usage: {litresPerPersonPerDay: number};
+    /**
+     * @description Whether the proposal includes rain water harvesting
+     */
+    rain: boolean;
+    /**
+     * @description Whether the proposal includes grey water re-use
+     */
+    grey: boolean;
+  };
+  /**
+   * @description Proposed energy sources
+   */
+  energy?: {
+    type: Array<'communityOwned' | 'heatPump' | 'solar'>;
```

Comment:
```suggestion
  /**
   * @uniqueItems true
   */
    type: Array<'communityOwned' | 'heatPump' | 'solar'>;
```

I think we should be able to uses this annotation? ([docs](https://json-schema.org/understanding-json-schema/reference/array#uniqueItems))